README.md

```diff
@@ -2,6 +2,12 @@ Kindly taken from https://exercism.org/tracks/elixir/exercises/tournament
 
 Full credits to problem definition and project setup to the original authors.
 
+# Refactor - Interview Notes
+
+1. Modify `CONTRIBUTING.md` or any similar developer setup instructions to refer to `.tool-versions` with [mise](https://mise.jdx.dev) or [asdf](https://asdf-vm.com)
```

Comment:
Essentially this lets most devs get started quickly with `asdf install` or `mise use`